content/blog/kubernetes-chart-v4/index.md

```diff
@@ -0,0 +1,1035 @@
+---
+title: "New: Helm Chart v4 resource with tons of new features"
+date: 2024-06-12T00:00:00-07:00
+meta_desc: >-
+  Pulumi Kubernetes v4.13 offers a new resource for applying Helm charts consistently across Pulumi SDKs, with new features and SDK support.
+
+authors:
+    - eron-wright
+
+tags:
+    - kubernetes
+    - yaml
+    - java
+    - helm
+---
+
+When you need to install a third-party application into your Kubernetes cluster, you're likely to find a
+Helm chart for that in [Artifact Hub](https://artifacthub.io/) or other registry. Pulumi provides two
+ways to apply a Helm chart, as outlined in [Choosing the Right Helm Resource For Your Use Case](/registry/packages/kubernetes/how-to-guides/choosing-the-right-helm-resource-for-your-use-case/).  The Chart resource offers deeper integration with Pulumi
+and better drift remediation.
+
+Today we're happy to announce a new "v4" version of the Chart resource, available now in v4.13 of the Pulumi Kubernetes provider.
+The new [kubernetes.helm.sh/v4.Chart](/registry/packages/kubernetes/api-docs/helm/v4/chart/) resource is provided
+side-by-side with the existing [kubernetes.helm.sh/v3.Chart](/registry/packages/kubernetes/api-docs/helm/v3/chart/) resource.
+We expect to deprecate v3 in the future.
+
+## What's New
```

Comment:
```suggestion
## What's new
```
Headings should use sentence case
https://github.com/pulumi/docs/blob/master/STYLE-GUIDE.md